.github/workflows/ci.yml

```diff
@@ -122,16 +127,16 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.8", "3.12"]
         db-backend: ["postgresql"]
         nautobot-version: ["stable"]
         include:
           - python-version: "3.11"
             db-backend: "postgresql"
             nautobot-version: "2.0.0"
-          # - python-version: "3.11"
```

Comment:
Review why mysql testing was disabled.

Reply:
A ticket was created to follow-up on this issue.

Reply:
> Review why mysql testing was disabled.

We recently tested this with another PR and the IPRange model doesn't work with MySQL. You can see it [here](https://github.com/nautobot/nautobot-app-firewall-models/actions/runs/9413334889/job/25929994388?pr=246).